pdb2pqr/__init__.py

```diff
@@ -1,21 +1,18 @@
-"""PDB2PQR
+"""PDB2PQR.
```

Comment:
Please remove the period from the end of the title, unless it's required by the formatting guidelines (in which case, we should discuss).  Thanks.

Reply:
https://docs.astral.sh/ruff/rules/ends-in-period/

It's because of this rule. Apparently the PEP, numpy and google style guides all follow this rule. If you don't like it, we can remove this rule.

Reply:
I removed the period and also added D400 as an ignored rule. Do not worry about it, I think it's quite useless too.

Anything else?